packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -50,9 +59,10 @@ export type SelectProps = {
    */
   isDisabled?: boolean;
   /**
-   * If `true`, the Select allows multiple selections
+   * @deprecated Use `hasMultipleChoices` instead.
    */
-  isMultiSelect?: boolean;
+  /** **Deprecated:** use `hasMultipleChoices` */
+  isMultiSelect?: HasMultipleChoices;
```

Comment:
@KevinGhadyani-Okta A [quick search](https://github.com/search?q=org%3Aokta%20isMultiSelect&type=code) doesn't indicate that `isMultiSelect` is in use anywhere. Do we want to just remove/rename the prop?

Reply:
All of our apps are in Atko, so you're actually seeing only a small portion of Okta. This is definitely used elsewhere :).

Reply:
Ah, yeah, that would make sense. :) Still learning my way around. Thanks for the context